docs/supported-os.md

```diff
@@ -0,0 +1,28 @@
+# Composer - Recommended OS versions
+
+The tables below provide recommended OS information for Composer 2.0+.
+
+## Windows
+
+OS                | Version            | Architectures
+------------------|--------------------|---------------
+Windows 10        | Version 1607+      | x32, x64
+Windows Server    | 2012 R2+           | x32, x64
+Nano Server       | Version 1803+      | x32, x64
+
+
+## macOS
+
+OS                | Version            | Architectures
+------------------|--------------------|---------------
+Mac OS X          | 10.14+             | x64
```

Comment:
should we note something about M1 (arm) version?

Reply:
Thanks @hatpick , added a sentence explicitly mentioning we don't run on M1 hardware.